Makefile

```diff
@@ -5,11 +5,14 @@ admin-build:
 	docker compose run --rm admin bash -c "npm --workspace @socialincome/admin install && npm run admin:build"
 
 admin-serve:
-	docker compose up admin
+	docker compose run --rm admin bash -c "npm --workspace=@socialincome/admin install && npm run admin:serve"
```

Comment:
If you use `run` you need to add the option `--service-ports`, otherwise, the ports to the container are not open.